### ISSUE TITLE
Parser URI not reset at reuse of model with additional query params.

### ISSUE THREAD

**Issue by MichMich** [robsontenorio/vue-api-query#92]
I'm running into an issue where the parser URI is not reset when I reuse a model with a query.

For example:
```js
const developers = Users.where('job', 'developer')

const page1 = await developers.limit(5).page(1).get()
// The url that is called is: /users?filter[job]=developer&limit=5&page=1

const page2 = await developers.limit(5).page(2).get()
// The url that is called is: /users?filter[job]=developer&limit=5&page=1&filter[job]=developer&limit=5&page=2
```
**Note that in the second request the query parameters are repeated!**

The desired result of the second request should of course be:
```js
const page2 = await developers.limit(5).page(2).get()
// The url that is called is: /users?filter[job]=developer&limit=5&page=2
```

When I manually reset the uri, the issue is gone:
```js
const developers = Users.where('job', 'developer')

const page1 = await developers.limit(5).page(1).get()
// The url that is called is: /users?filter[job]=developer&limit=5&page=1

developers._builder.parser.uri = ''

const page2 = await developers.limit(5).page(2).get()
// The url that is called is: /users?filter[job]=developer&limit=5&page=2
```

This seems like a bug, but maybe I missed something in the documentation.

**Comment by robsontenorio** [comment]
What do you intend to achieve with page 1 and page 2 variables?

Pagination should not be handled by this way. Please, check the readme.

**Comment by MichMich** [comment]
It is just an example. Not specific to pagination. The reason I ran into it is because I passed a model (with additional queries) to a Vue component: 
```HTML
<user-list :model=“User.where(‘job’, ‘developer’)” />
```

If I use pagination or any other additional queries on that model prop from within the Vue component, the attributes get duplication when I reuse the model. 

Hope this makes sense.

**Comment by MichMich** [comment]
I think a simple solution would be to reset the `uri` before line 16 of the parser:  https://github.com/robsontenorio/vue-api-query/blob/master/src/Parser.js#L16

```js
this.uri = ‘’
```

**Comment by MichMich** [comment]
If you want I can send a PR.

**Comment by robsontenorio** [comment]
Please, consider submit tests when sending a PR. Thanks!

**Comment by robsontenorio** [comment]
Closed by #106
